kategory-docs/docs/docs/datatypes/try/README.md

```diff
@@ -4,6 +4,171 @@ title: Try
 permalink: /docs/datatypes/try/
 ---
 
-## Try 
+## Try
+
+Kategory has a lots of different types of error handling and reporting, which can make it difficult to decide which one is best for your situation. 
```

Comment:
Link to http://kategory.io/docs/patterns/error_handling/ for disambiguation on when to use each strategy

Reply:
https://github.com/kategory/kategory/pull/458/commits/2284da888d63c7efc4ef0587b2ae06e7a60486b5